.gitignore

```diff
@@ -16,3 +16,4 @@ data
 *.bak
 *~
 *.vscode
+src/descriptors/validation_config.yml
```

Comment:
Why is this file being ignored? I believe we need it in the repo.

Reply:
We don't need it in the repo.  It is generated from power_system_structs.json, which is in the repo.  It seems unnecessary to store the same information somewhere else.  Kyra currently has it set to get built during build().